Set default path to root URL

madsleegiil opened the pull request:
This PR sets "/" as default path when removing cookies. 
Please see [issue 1257](https://github.com/tipsy/javalin/issues/1257) for details.

A new test class has been added as I could not see any existing test class where the test cases would fit.

tipsy commented:
Great work @madsleegiil! There are a lot of cookie tests in `TestRequest` and `TestResponse`, it would be nice if you could also move them to `TestCookies`

madsleegiil commented:
> Great work @madsleegiil! There are a lot of cookie tests in `TestRequest` and `TestResponse`, it would be nice if you could also move them to `TestCookies`

I have moved the test cases that I thought was relevant. Please point out if any more are missing. 
I also corrected the tests I wrote as they did not considered the path when asserting on removed cookies.

tipsy commented:
Nice! Could you please revert the accidental formatting changes?

madsleegiil commented:
Sorry for not having been more careful with the automatic formatting changes. 
Will the PR be squash merged if approved? If not, do you prefer that I clean up the commit history on the branch myself?

tipsy commented:
> Sorry for not having been more careful with the automatic formatting changes.

Not problem at all. The change looks great! 

> Will the PR be squash merged if approved? If not, do you prefer that I clean up the commit history on the branch myself?

You don't have to clean anything up, I have this button: 
![image](https://user-images.githubusercontent.com/1521451/120205703-8e7e7800-c22a-11eb-9219-d792b82688ee.png)

I'll click it once the tests pass :)
